frontend/src/common/unixToFullDate.ts

```diff
@@ -7,7 +7,7 @@ import { months } from './dates'
  */
 export const unixToFullDate = (unixDate: number) => {
   const date = new Date(unixDate * 1000)
-  const month = date.getMonth() + 1
+  const month = date.getMonth()
```

Comment:
do check with the author of this file? since this causes different outputs

Reply:
can verify this is correct, cos `date.getMonth()` gives the month index (i.e. January = 1), and the function use the month index to get the month text `${months[month]}`, and `months` is a list of month name strings with 'January' at index 0. So this function would function as intended for all use cases,